demo/index.html

```diff
@@ -29,15 +29,13 @@
   </div>
 
   <script>
-
-    Polymer({
-
-      is: 'meta-test',
-
-      ready: function() {
-        this.textContent = new Polymer.IronMetaQuery({key: 'info'}).value;
-      }
-
+    document.addEventListener('WebComponentsReady', function() {
```

Comment:
~~[Should we use `window`?](http://webcomponents.org/polyfills/html-imports/#the-webcomponentsready-event)~~ [Looks like we don't need to.](https://github.com/webcomponents/webcomponentsjs/blob/b7d7f3cc35999e8e588f7f31e2bfb7a1f77bb097/src/CustomElements/boot.js#L99) This is something I've always been confused about. Are there any events which are fired on `window` but not `document`? (Or non-bubbling events on `document` for that matter?)